repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/stock/ResponseBytesClfHandler.java

```diff
@@ -19,19 +19,34 @@
  */
 package org.openrepose.commons.utils.logging.apache.format.stock;
 
+import org.openrepose.commons.utils.StringUtilities;
```

Comment:
We should make a story to replace all uses of `org.openrepose.commons.utils.StringUtilities` with `org.apache.commons.lang3.StringUtils`.